src/validation/fields.ts

```diff
@@ -95,6 +95,32 @@ export const redirectTo = Joi.string()
       return value;
     }
 
+    // * Check if the redirectTo is a deep link: start with any sequence of characters
+    // * followed by "://", and then followed by any sequence of characters
+    // * we skip values that start with http: or https://
+    if (/^(?!https?:\/\/).+:\/\/.*/.test(value)) {
```

Comment:
what if the user configures:

```
myapp://redirect
```

but the redirecTo is:

```
myapp://somethingelse
```

?

Reply:
It will be rejected. Good catch 👍 , will adjust